sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -847,6 +848,11 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
     sql("DROP TEMPORARY FUNCTION udtf_count2")
   }
 
+  test("SPARK-30868 ADD JAR command failed") {
+    val jarPath = "file:///tmp/not_exists.jar"
```

Comment:
can we use some common commands like `CREATE TABLE`, `SELECT`, `INSERT`, etc.?

Reply:
sure